script/testing/util/common.py

```diff
@@ -6,13 +6,17 @@
 import re
 import signal
 import errno
-from util.constants import LOG
+import psutil
+import datetime
+from util.constants import LOG, FILE_CHECK_PID_EXISTS, FILE_KILL_SERVER_ON_PORT, CommandLineStr, ErrorCode
```

Comment:
This is messy. Just do `import util.constants`.

Reply:
fixed